src/components/forms/fragments/exchangeApplication/ExchangeApplication.tsx

```diff
@@ -79,10 +79,9 @@ const ExchangeApplication: FC<ExchangeApplicationProps> = ({ berths, reasons })
       <Row>
         <Col>
           <Select name="reason">
-            <FormattedMessage
-              tagName="option"
-              id="page.berth.exchange_application.reason.default"
-            />
+            <FormattedMessage id="page.berth.exchange_application.reason.default">
+              {id => <option value="">{id}</option>}
```

Comment:
you should have option wrap formattedMessage inside instead

Reply:
But the thing is `FormattedMessage` renders a `span` by default, and here we need only plain text inside the `option`.
So in your example above, it renders:
```html
<option value="">
  <span>foo default</span>
</option>
```

Reply:
On the other hand,
```html
<FormattedMessage id="page.berth.exchange_application.reason.default">
   {id => <option value="">{id}</option>}
</FormattedMessage>
```
renders
```html
<option value="">
   foo default
</option>
```

Reply:
But if there are no `id`, it will return a span instead of option inside of `select`, that's not good